src/Eris/Generator/FrequencyGenerator.php

```diff
@@ -16,6 +16,7 @@ function frequency(/*$frequencyAndGenerator, $frequencyAndGenerator, ...*/)
 class FrequencyGenerator implements Generator
 {
     private $generators;
+    private $frequencies;
```

Comment:
@giorgiosironi 

Noticed this myself, figured it would be out of scope of this PR and wanted to follow up, what do you think?